gravitee-management-api-model/src/main/java/io/gravitee/management/model/parameters/Key.java

```diff
@@ -39,6 +39,9 @@
     PORTAL_APIS_SHOW_TAGS_IN_APIHEADER("portal.apis.apiheader.showtags.enabled", "false"),
     PORTAL_APIS_SHOW_VIEWS_IN_APIHEADER("portal.apis.apiheader.showviews.enabled", "false"),
     PORTAL_DASHBOARD_WIDGETS("portal.dashboard.widgets"),
+    PORTAL_UPLOAD_IMAGES_ENABLED("portal.uploadImages.enabled", "false"),
```

Comment:
I think it could be better to rename IMAGES to MEDIA

Reply:
done